src/main/java/jp/pay/model/Charge.java

```diff
@@ -309,4 +319,17 @@ public Charge capture(Map<String, Object> params, RequestOptions options)
 		return request(RequestMethod.POST, String.format("%s/capture",
 						instanceURL(Charge.class, this.getId())), params, Charge.class, options);
 	}
+
+	public Charge tdsFinish(Map<String, Object> params, RequestOptions options)
+			throws AuthenticationException, InvalidRequestException,
+			APIConnectionException, CardException, APIException {
+		return request(RequestMethod.POST, String.format("%s/tds_finish", instanceURL(Charge.class, this.getId())), params, Charge.class, options);
```

Comment:
下のstatic method、他のPOST系エンドポイントではないこともあるのでなくしても良さそうですが、残すなら実装はstatic側だけにしたほうがメンテも楽かなと。

params不要だし、引数からなくしてもいいと思うのですが、全エンドポイントのインターフェースがそうなってる感じもあるので、残すならstatic側にもつけていいかも。
（他見るとoverloadとか多くて本当にそんなたくさん必要？って思うけれど）

```suggestion
		return tdsFinish(this.getId(), options);
```

Reply:
paramsは残しといた方が今後引数が増えた時にも使えるので汎用的な気がしますね。
staticなメソッドが増えるのは嬉しくないんですがインスタンス作らないと呼べないってのも困るので、staticだけにした方がいいかな